### PR TITLE
Make repo ready for CNCF

### PR DESCRIPTION
**Pull request by WanzenBug**
Adds:
* Code of conduct
* Maintainers list
* Contributing guide
* governance policy

Changed:
* moved yaml deployment to `/legacy`
* updated README to point to Piraeus Operator deployment

Most of the "paperwork" was inspired by https://github.com/openkruise/kruise/

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# The Piraeus Datastore Maintainers
+
+This file lists the maintainers of the entire Piraeus Datastore project, hosted and managed by the
+[Piraeus Datastore Organization](https://github.com/piraeusdatastore). The responsibilities of maintainers are listed in
+the [GOVERNANCE.md](./GOVERNANCE.md) file
+
+## Maintainers
```
Comment (rck) [inline MAINTAINERS.md:7]: maybe add some "in alphabetical order" to make clear this does not imply anything else than that.

**Review by rck — CHANGES_REQUESTED** [review]

**Comment by rck** [comment]
while these are established standards, do we want some S-o-b from all that will be maintainers? Quite frankly I think this would be a good idea in this case.

**Review by rck — APPROVED** [review]

**Comment by rck** [comment]
LGTM, feel free to eventually add `Signed-off-by: Roland Kammerer <roland.kammerer@linbit.com>`

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# Contributing to Piraeus
+
+Welcome to Piraeus! Piraeus consists several repositories under the organization.
```
Comment (JoelColledge) [inline CONTRIBUTING.md:3]:
```suggestion
Welcome to Piraeus! Piraeus consists of several repositories under the organization.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+network locations, private business data and so on.
+
+## Code and doc contribution
+
+Every action to make Piraeus better is encouraged.
+On GitHub, every improvement for Piraeus could be via a PR (short for pull request).
+
+- If you find a typo, try to fix it!
+- If you find a bug, try to fix it!
+- If you find some redundant codes, try to remove them!
```
Comment (JoelColledge) [inline CONTRIBUTING.md:37]:
```suggestion
- If you find some redundant code, try to remove it
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+please remember to remove sensitive data from your issue description. Sensitive data could be password, secret key,
+network locations, private business data and so on.
+
+## Code and doc contribution
+
+Every action to make Piraeus better is encouraged.
+On GitHub, every improvement for Piraeus could be via a PR (short for pull request).
+
+- If you find a typo, try to fix it!
```
Comment (JoelColledge) [inline CONTRIBUTING.md:35]: "!" on every point are not really English style.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+- If you find a bug, try to fix it!
+- If you find some redundant codes, try to remove them!
+- If you find some test cases missing, try to add them!
+- If you could enhance a feature, please DO NOT hesitate!
+- If you find code implicit, try to add comments to make it clear!
+- If you find code ugly, try to refactor that!
+- If you can help to improve documents, it could not be better!
```
Comment (JoelColledge) [inline CONTRIBUTING.md:42]:
```suggestion
- If you can help to improve documentation, that is great
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+- If you find document incorrect, just do it and fix that!
```
Comment (JoelColledge) [inline CONTRIBUTING.md:43]:
```suggestion
- If you find documentation incorrect, go ahead and fix it
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+- performance issues
+- feature proposal
+- feature design
+- help wanted
+- doc incomplete
+- test improvement
+- any questions on project
+- and more...
+
+please remember to remove sensitive data from your issue description. Sensitive data could be password, secret key,
```
Comment (JoelColledge) [inline CONTRIBUTING.md:27]:
```suggestion
Please remember to remove sensitive data from your issue description. Sensitive data includes passwords, secret keys,
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+request changes or approve of the request as is. Once approved, the maintainer will merge the changes into the Piraeus
+repository.
+
+## Engage to help anything
+
+We choose GitHub as the primary place for Piraeus to collaborate.
+So the latest updates of Piraeus are always here.
+Although contributions via PR is an explicit way to help, we still call for any other ways.
+
+- reply to other's issues if you could;
```
Comment (JoelColledge) [inline CONTRIBUTING.md:65]:
```suggestion
- reply to issues if you can;
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+- help solve other user's problems;
```
Comment (JoelColledge) [inline CONTRIBUTING.md:66]:
```suggestion
- help solve other users' problems;
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,94 @@
...
+- help review other's PR design;
```
Comment (JoelColledge) [inline CONTRIBUTING.md:67]:
```suggestion
```

I don't what the difference is between this point and the next one.

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+- help review other's codes in PR;
```
Comment (JoelColledge) [inline CONTRIBUTING.md:68]:
```suggestion
- help review the code from other contributors in PRs;
```

File: CONTRIBUTING.md [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+- discuss about Piraeus to make things clearer;
+- advocate Piraeus technology beyond GitHub;
+- write blogs on Piraeus and so on.
+
+In a word, **ANY HELP IS CONTRIBUTION**.
+
+## Join Piraeus as a member
+
+It is also welcomed to join Piraeus team if you are willing to participate in Piraeus community continuously and keep active.
```
Comment (JoelColledge) [inline CONTRIBUTING.md:77]:
```suggestion
It is also welcomed that you join the Piraeus team if you are willing to participate in the Piraeus community regularly.
```

File: CONTRIBUTING.md [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+
+### Requirements
+
+- Have read [Contributing to Piraeus](./CONTRIBUTING.md) carefully
+- Have read [Contributor Covenant Code of Conduct](./CODE_OF_CONDUCT.md)
+- Have submitted multi PRs to the community
```
Comment (JoelColledge) [inline CONTRIBUTING.md:83]:
```suggestion
- Have submitted multiple PRs to the community
```

File: CONTRIBUTING.md [hunk 14]
```diff
@@ -0,0 +1,94 @@
...
+- Be active in the community, may including but not limited
```
Comment (JoelColledge) [inline CONTRIBUTING.md:84]:
```suggestion
- Be active in the community; may include but not limited to
```

File: CONTRIBUTING.md [hunk 15]
```diff
@@ -0,0 +1,94 @@
...
+    - Submitting or commenting on issues
+    - Contributing PRs to the community
+    - Reviewing PRs in the community
+
+### How to do it
+
+You can do it in either of two ways:
+
+- Submit a PR in the project repo
+- Contact via the [community](./README.md#community) channels offline
```
Comment (JoelColledge) [inline CONTRIBUTING.md:94]:
```suggestion
- Contact the team via the [community](./README.md#community) channels
```

File: MAINTAINERS.md [hunk 16]
```diff
@@ -0,0 +1,16 @@
...
+
+in alphabetical order:
```
Comment (JoelColledge) [inline MAINTAINERS.md:9]:
```suggestion
In alphabetical order:
```

**Review by JoelColledge — APPROVED** [review]
Looks good. You're welcome to add `Signed-off-by: Joel Colledge <joel.colledge@linbit.com>` to the commit message too.

**Pull request merged**